conda/recipes/libcuspatial/build.sh

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
+
+# This assumes the script is executed from the root of the repo directory
+# show environment
+printenv
+# build cuspatial with verbose output
+cd $WORKSPACE
+echo "Fix permissions on build script"
+chmod ugo+x ./build.sh
```

Comment:
Why is this needed? Should the permissions be fixed on the file and then committed to the repo instead?